packages/react/src/Bento/Bento.module.css

```diff
@@ -0,0 +1,1793 @@
+  grid-column-start: 6;
+}
+
+.Bento__Item--column-start-7 {
+  grid-column-start: 7;
+}
+
+.Bento__Item--column-start-8 {
+  grid-column-start: 8;
+}
+
+.Bento__Item--column-start-9 {
+  grid-column-start: 9;
+}
+
+.Bento__Item--column-start-10 {
+  grid-column-start: 10;
+}
+
+.Bento__Item--column-start-11 {
+  grid-column-start: 11;
+}
+
+.Bento__Item--column-start-12 {
+  grid-column-start: 12;
+}
+
+/* Row Start */
+.Bento__Item--row-start-1 {
+  grid-row-start: 1;
+}
+
+.Bento__Item--row-start-2 {
+  grid-row-start: 2;
+}
+
+.Bento__Item--row-start-3 {
+  grid-row-start: 3;
+}
+
+.Bento__Item--row-start-4 {
+  grid-row-start: 4;
+}
+
+.Bento__Item--row-start-5 {
+  grid-row-start: 5;
+}
+
+.Bento__Item--row-start-6 {
+  grid-row-start: 6;
+}
+
+.Bento__Item--row-start-7 {
+  grid-row-start: 7;
+}
+
+.Bento__Item--row-start-8 {
+  grid-row-start: 8;
+}
+
+.Bento__Item--row-start-9 {
+  grid-row-start: 9;
+}
+
+.Bento__Item--row-start-10 {
+  grid-row-start: 10;
+}
+
+.Bento__Item--row-start-11 {
+  grid-row-start: 11;
+}
+
+.Bento__Item--row-start-12 {
+  grid-row-start: 12;
+}
+
+/* Bento Item Vertical Align */
+.Bento__Item--verticalAlign-start {
+  justify-content: start;
+}
+
+.Bento__Item--verticalAlign-center {
+  justify-content: center;
+}
+
+.Bento__Item--verticalAlign-end {
+  justify-content: end;
+}
+
+/* Bento Item Horizontal Align */
+.Bento__Item--horizontalAlign-start {
+  align-items: start;
+}
+
+.Bento__Item--horizontalAlign-center {
+  align-items: center;
+}
+
+.Bento__Item--horizontalAlign-end {
+  align-items: end;
+}
+
+/* Bento Visual */
+.Bento__Visual {
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+}
+
+.Bento__Item--visual-as-background .Bento__Visual {
+  order: -1;
+  grid-row: 1 /-1;
+  grid-column: 1 /-1;
+}
+
+.Bento__Visual > img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+
+.Bento__Visual-no-fill > img {
+  height: 100%;
+  width: 100%;
+  object-fit: contain;
+  border-radius: 0px;
+}
+
+/* Bento Padding */
+.Bento-padding--condensed {
+  padding: var(--brand-Bento-Content-padding-condensed);
+}
+.Bento-padding--normal {
+  padding: var(--brand-Bento-Content-padding-normal);
+}
+.Bento-padding--spacious {
+  padding: var(--brand-Bento-Content-padding-spacious);
+}
+
+.Bento-padding--condensed > img {
+  border-radius: var(--brand-Bento-Item-borderRadius-small);
+}
+.Bento-padding--normal > img {
+  border-radius: var(--brand-Bento-Item-borderRadius-small);
+}
+.Bento-padding--spacious > img {
+  border-radius: var(--brand-Bento-Item-borderRadius-small);
+}
+
+@media screen and (min-width: 80rem) {
+  .Bento-padding--condensed > img {
+    border-radius: var(--brand-Bento-Item-borderRadius-large);
+  }
+  .Bento-padding--normal > img {
+    border-radius: var(--brand-Bento-Item-borderRadius-large);
+  }
+  .Bento-padding--spacious > img {
+    border-radius: var(--brand-Bento-Item-borderRadius-large);
+  }
+}
+
+@media screen and (min-width: 48rem) {
```

Comment:
Just for ease of reading, could you put the lower breakpoint queries above the higher ones? Especially because we're using min-width here.

Reply:
Grid has this reversed as well. I will update both to use this format instead.

Reply:
Or we can include that in another PR but it should be updated in the same way.

Reply:
I found that this broke the working state for both of these components, so I didn't include this update.